backend/climateconnect_main/celery.py

```diff
@@ -3,20 +3,25 @@
 from celery import Celery
 from celery.schedules import crontab
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'climateconnect_main.settings')
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "climateconnect_main.settings")
 
-app = Celery('climateconnect_main')
+app = Celery("climateconnect_main")
 
 # Using a string here means the worker doesn't have to serialize
 # the configuration object to child processes.
 # - namespace='CELERY' means all celery-related configuration keys
 #   should have a `CELERY_` prefix.
-app.config_from_object('django.conf:settings', namespace='CELERY')
+app.config_from_object("django.conf:settings", namespace="CELERY")
 app.autodiscover_tasks()
 
+# add weekly scheduling
 app.conf.beat_schedule = {
     'schedule_automated_email_reminder_for_notifications': {
         'task': 'climateconnect_api.tasks.schedule_automated_reminder_for_user_notifications',
         'schedule': crontab(minute=0, hour=0)
-    }
+    },
+    'schedule_weekly_recommendations_email': {
+        'task': 'climateconnect_api.tasks.schedule_weekly_recommendations_email',
+        'schedule': crontab(minute=11, hour=11, day_of_week=5)
+    },
```

Comment:
@atlas033 since the celery issue is not fixed yet can you put this code under comments?  Both jobs
`schedule_weekly_recommendations_email` and `schedule_weekly_recommendations_email`